dapps/digital-twin-detail/src/components/sharing/SharingOverview.vue

```diff
@@ -20,106 +20,117 @@
 <template>
   <div class="content pt-5">
     <div
-      class="d-flex flex-row justify-content-between align-items-center"
-      style="max-height: 33px"
+      v-if="!$store.state.twin.isOwner"
+      class="text-center d-flex flex-column justify-content-center align-items-center vh-100"
     >
-      <div class="search">
-        <label
-          id="twin-enable-search"
-          for="searchInput"
-          @click="isActiveSearch = true"
-        >
-          <i
-            class="mdi mdi-magnify mr-1"
-            style="font-size: 22px"
+      <h1>
+        <i class="mdi mdi-eye-off-outline" />
+      </h1>
+      <span>{{ '_evan.no-permission' | translate }}</span>
```

Comment:
Translations for `_evan.no-permission` are missing

Reply:
The translation comes from one of Karl's PRs :)